### ISSUE TITLE
Deno fmt command does not complain about missing files

### ISSUE THREAD

**Issue by srackham** [denoland/deno#3915]
```
$ deno fmt non-existant-file.ts
Formatted 0 files in 213ns

$ echo $?
0

$ deno --version
deno 0.32.0
v8 8.1.108
typescript 3.7.2
```
Is this the expected behaviour?

**Comment by nayeemrmn** [comment]
Maybe we should handle this like the test runner; non-zero exit code if there are no matching files.

**Comment by srackham** [comment]
@nayeemrmn: That would still allow file name typos to slip through unnoticed e.g.

```
deno fmt non-existent-file.ts existing-file.ts  # 1 file formatted.

**Comment by kevinkassimo** [comment]
I think a basic solution is simply to warn "No file matching {...}" when we get empty paths after running glob on a pattern

**Comment by bartlomieju** [comment]
Fixed in #3988 and released in `v0.33.0`
